package.json

```diff
@@ -73,6 +72,7 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react-native": "^4.0.0",
     "codecov": "^3.0.2",
```

Comment:
Does this replace codecov or are we using both?

Reply:
The intent was to replace Codecov with Coveralls but I forgot to remove the old dependency. Updated now.